src/Microsoft.DocAsCode.Build.ManagedReference/FillReferenceInformation.cs

```diff
@@ -26,6 +27,7 @@ public class FillReferenceInformation : BaseDocumentBuildStep, ICanTraceContextI
 
         public override void Postbuild(ImmutableList<FileModel> models, IHostService host)
```

Comment:
>Postbuild [](start = 29, length = 9)

So PostBuild is safe, but for "build", if context is changed, build step should guarantee thread safety?